backend/src/main/java/com/redhat/cloud/notifications/routers/EventResource.java

```diff
@@ -59,12 +60,8 @@ public Page<EventLogEntry> getEvents(@Context SecurityContext securityContext, @
                                          @RestQuery String eventTypeDisplayName, @RestQuery LocalDate startDate, @RestQuery LocalDate endDate,
                                          @RestQuery Set<String> endpointTypes, @RestQuery Set<Boolean> invocationResults,
                                          @RestQuery Set<EventLogEntryActionStatus> status,
-                                         @BeanParam Query query,
+                                         @BeanParam @Valid Query query,
                                          @RestQuery boolean includeDetails, @RestQuery boolean includePayload, @RestQuery boolean includeActions) {
-        if (query.getLimit().getLimit() < 1 || query.getLimit().getLimit() > 200) {
-            throw new BadRequestException("Invalid 'limit' query parameter, its value must be between 1 and 200");
```

Comment:
Please do not lower the max limit for this endpoint. It has to keep returning 200 entries because other ConsoleDot apps depend on it.

Reply:
Oh, sorry about that. I will revert the `pageSize` back to 200.